propagation.go

```diff
@@ -169,6 +164,107 @@ func (p *textMapPropagator) Extract(abstractCarrier interface{}) (SpanContext, e
 	return ctx, nil
 }
 
+type zipkinB3Propagator struct {
```

Comment:
Preferably `./zipkin/propagation.go` (the other zipkin.go is something else, but we can move it under `./zipkin/...` later too.